test/gotoTest.test.ts

```diff
@@ -0,0 +1,181 @@
+import * as assert from "assert";
```

Comment:
We may want to run unit test in Travis CI.

Reply:
Sounds like a plan!

Reply:
Sure! no need to do it in this PR.